x-pack/plugin/core/src/test/java/org/elasticsearch/test/SecuritySettingsSourceField.java

```diff
@@ -13,5 +13,16 @@
     public static final String TEST_PASSWORD = "x-pack-test-password";
     public static final String TEST_INVALID_PASSWORD = "invalid-test-password";
 
+    public static final String UNRESTRICTED_ROLE = "_es_test_root";
+    public static final String UNRESTRICTED_ROLE_YML = """
+        _es_test_root:
+          cluster: [ ALL ]
+          indices:
+            - names: '*'
+              allow_restricted_indices: true
+              privileges: [ ALL ]
+          run_as: [ '*' ]
```

Comment:
I assume you just mean "have the same definition" not "read it the definition from `build-tools/src`

Reply:
Yes same definition, not "reading from the same file"

Reply:
It turn out this isn't really possible (at least not without other changes).

Some of the `testclusters` tests rely on application privileges, so that definition currently needs the have them. I'd like to remove application privileges from this role, but it would require changing some tests (which I'd like to avoid in this PR).

However, adding those application privileges to `SingleNodeTestCase` doesn't work because it causes role resolution to try and load application privileges, which isn't possible while we're trying to wipe the security index between test executions.

The longer term fix is to remove all application privileges from `_es_test_root` in `testclusters`, but I don't think I can tackle that here.

Reply:
The same is not an issue for `testclusters`? Doesn't `testclusters` also wipe security indices between tests?

Reply:
It isn't a problem.
I think it's because the Rest API works fine, but `SingleNodeTestCase` does a bunch of intrusive things that aren't really supported

Reply:
OK. I think it's ok to leave the root user for `testclusters` as is. It is at least useful for `./gradlew :run` to allow an all powerful user for manual testings.